src/code_ddaf0_len_c50.c

```diff
@@ -1,22 +1,12 @@
 #include "common.h"
 
-typedef struct {
-    /* 0x00 */ u8 unk_00;
-    /* 0x04 */ s32 unk_04;
-    /* 0x08 */ s32 unk_08;
-    /* 0x0C */ char unk_01[0x10];
-    /* 0x1C */ u8 unk_1C;
-} struct_80159D50; // size = 0x20
-
-extern struct_80159D50 D_80159D50[0x40];
-
+// this is definitely wrong but I had issues trying to use an array accessor
```

Comment:
Nah, I came to the conclusion that they tracked it separately as well. It's kinda neat that gcc can make that distinction.

Reply:
Even if they're incrementing the struct pointer and using that, I still think something about this is wrong. What do you mean about gcc making a distinction?